examples/Cargo.toml

```diff
@@ -14,6 +15,7 @@ members = [
 ]
 
 [workspace.dependencies]
+alloy = { git = "https://github.com/alloy-rs/alloy", rev = "b79db21734cffddc11753fe62ba571565c896f42", default-features = false }
```

Comment:
Please add a comment why we need this specific revision (as opposed to some version).